encoding.go

```diff
@@ -86,12 +86,12 @@ func parseLeafNode(serialized []byte, depth byte, comm []byte) (VerkleNode, erro
 	if NodeWidth != len(values) {
```

Comment:
All these are doing two things at once:
- Avoiding more magic numbers
- Indirectly the `SerializedAffinePointSize` has been switched from 32 to 64, since now both points are part of the serialization.